### ISSUE TITLE
Zowe CLI V2 - Grouping of profiles in the same LPAR declaration.

### ISSUE THREAD

**Issue by FALLAI-Denis** [zowe/zowe-cli#1406]
Hi,

According to the documentation, (which is unclear), a profile can be declared in "1st level" with an independent management according the targeted LPAR, or else can be declared in "2nd level" inside a "1st level" LPAR declaration, to share common properties at the LPAR level.

But this is to be managed by hand by directly editing the config files: **it's not acceptable.**

Config files should not be managed directly under editor. It is imperative to provide zowe commands to manage the profiles, which is the only way to avoid handling errors.

**Comment by FALLAI-Denis** [comment]
Confer #1413

**Comment by JirkaAichler** [comment]
I agree that profile management in v2 is not intuitive at all. I would also welcome some simplification to set up my profiles easily. I was hoping that the `zowe config init` command will help me. But it only generated a not working configuration that I had to tweak manually.
